src/main/charts/bamboo/values.yaml

```
@@ -340,7 +340,67 @@ ingress:
 # Bamboo configuration 
 #
 bamboo:
-  
+
+  # -- Override the server/agent broker URL; this is optional.
+  brokerUrl:
+
+  # The security token with which the agent will authenticate to the Bamboo server
+  # Additional details here:
+  # https://confluence.atlassian.com/bamboo/agent-authentication-289277196.html#Agentauthentication-SecuritytokenverificationSecuritytokenverification
+  #
+  securityToken:
+    # -- The name of the K8s Secret that contains the security token. When specified the token 
+    # will overrided the generated one. This secret should also be shared with the agent deployment.
+    # An Example of creating a K8s secret for the secret below:
+    # 'kubectl create secret generic <secret-name> --from-literal=security-token=<security token>'
+    # https://kubernetes.io/docs/concepts/configuration/secret/#opaque-secrets
+    #
+    secretName:
+
+    # -- The key (default `secretKey`) in the Secret used to store the Bamboo shared key.
+    #
+    secretKey: security-token
+
+  # -- The license to provide to the Bamboo nodes (optional). If you
+  # have an existing license it can be provided to the server up-front
+  # to skip the configuration screen on first run.
+  #
+  license:
+
+    # -- The secret that contains the license information
+    #
+    secretName:
+
+    # -- The key (default 'licenseKey') in the Secret used to store the license information
+    #
+    licenseKey: license
```

```suggestion
    secretKey: license
```

The other products are using `secretKey` for the license.
https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/jira/values.yaml#L470